provider/policy/resource/general.py

```diff
@@ -0,0 +1,71 @@
+from typing import List
+
+from shared.common import BaseOptions
+from shared.common import (
+    ResourceProvider,
+    Resource,
+    message_handler,
+    ResourceDigest,
+    ResourceEdge,
+)
+from shared.error_handler import exception
+
+
+class IamUser(ResourceProvider):
+    def __init__(self, options: BaseOptions):
+        """
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [One-line docstring should fit on one line with quotes (found 4) (D200)](https://app.codacy.com/gh/Cloud-Architects/cloud-discovery/pullRequest?prid=5655786)